setup/q2rtx.sh

```diff
@@ -2,21 +2,28 @@
 
 BIN_PREFIX="."
 
+XDG_DATA_HOME := ${HOME}/.local/share
+
 # If the game is installed via a package manager q2rtx won't be in the same
 # directory as q2rtx.sh
 if [[ -d "/usr/share/quake2rtx" ]]; then
 	BIN_PREFIX="/usr/share/quake2rtx/bin"
 fi
 
+if [[ -d "${HOME}/.quake2rtx" ]]; then
+        mv "${HOME}/.quake2rtx" "${XDG_DATA_HOME}/quake2rtx"
+fi
```

Comment:
This opens a can of worms for anyone switching between old and new versions of q2rtx. Running an old copy after this mv will recreate ~/.quake2rtx, and then running a new version with this change will end up moving the old directory into a subdirectory of the new directory.

At the very least, this change should check whether `${XDG_DATA_HOME}/quake2rtx` exists before moving the old directory. But this will still create a mess for anyone who ever downgrades across this change.

Reply:
Fixed in 41ee020bdafcd931e61435d6314ff2ac27fc8202.